restapi/src/test/java/io/stargate/web/docsapi/service/json/JsonConverterTest.java

```diff
@@ -1,130 +0,0 @@
-package io.stargate.web.docsapi.service.json;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.node.BooleanNode;
-import com.fasterxml.jackson.databind.node.ObjectNode;
-import io.stargate.db.datastore.Row;
-import io.stargate.web.docsapi.service.DocumentServiceTest;
-import java.util.*;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-
-public class JsonConverterTest {
```

Comment:
Why does this change show up as a delete + add as opposed to a file move? Was there a lot of code changes in this file?

Reply:
I did add unit tests, but other than that there weren't changes...so I'm not sure